src/EnergyPlus/HWBaseboardRadiator.hh

```diff
@@ -174,9 +192,22 @@ namespace HWBaseboardRadiator {
         }
     };
 
+    struct HWBaseboardDesignNumericFieldData
+    {
+        // Members
+        Array1D_string FieldNames;
+
+        // Default Constructor
+        HWBaseboardDesignNumericFieldData()
+        {
+        }
+    };
+
     // Object Data
     extern Array1D<HWBaseboardParams> HWBaseboard;
+    extern Array1D<HWBaseboardDesignData> HWBaseboardDesignObject;
     extern Array1D<HWBaseboardNumericFieldData> HWBaseboardNumericFields;
+    extern Array1D<HWBaseboardDesignNumericFieldData> HWBaseboardDesignNumericFields;
```

Comment:
We should follow this up quickly with a PR that moves this to the deglobalized state, but not for today obviously.